nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/ITestHandleHttpRequest.java

```diff
@@ -273,7 +276,8 @@ public void run() {
         mff.assertAttributeExists("http.headers.multipart.content-disposition");
     }
 
-    @Test(timeout = 30000)
+    @Test
+    @Timeout(value = 30000, unit = TimeUnit.MILLISECONDS)
```

Comment:
This can be changed to `@Timeout(30)` and the `TimeUnit` can be removed.

Reply:
My bad I missed that one, making the changes!